### ISSUE TITLE
Error occurred, please select a different payment method or try again later.

### ISSUE THREAD

**Issue by crtxz** [Cloudstek/whmcs-mollie-payment-gateway#6]
Hello,

I get the following error at the invoice: Error occurred, please select a different payment method or try again later.

I really don't know how it's working.

**Comment by mdeboer** [comment]
I need more information to know what's going on. Please turn on module logging in WHMCS and post any relevant log entries here so I can take a look.

**Comment by crtxz** [comment]
This module isn't being logged. I have turned it on but there isn't info at all coming from this module.

**Comment by mdeboer** [comment]
Hmm odd, that usually indicates something low level is wrong. Have you tried turning on PHP error logging to see if you can get any output at all? It's guesswork at best without any further info.

Also, what PHP version are you using and what version of WHMCS?

**Comment by crtxz** [comment]
Hi,

PHP Version 7.0.27
WHMCS 7.4.2

There aren't any errors, that is the whole problem with solving this

**Comment by mdeboer** [comment]
Okay, well it's been a while since I touched the code (I stopped developing for WHMCS for many reasons). With a quick glance on the code it looks like exceptions from the Mollie API aren't logged but are only handled by a generic error message. I will release a patch today which I would like you to test to fix this.

Sorry for the inconvenience!

**Comment by crtxz** [comment]
Thanks for considering updating the code. Did you already update it so I can test it?

**Comment by crtxz** [comment]
Any update regarding this?

**Comment by mdeboer** [comment]
Nope sorry, work got in the way but it's on my list 👍

**Comment by crtxz** [comment]
Any update? :)

**Comment by mdeboer** [comment]
Working on it right now 👍

**Comment by mdeboer** [comment]
Okay as I can't test myself, would you be so kind to remove the module and install this release: https://github.com/Cloudstek/whmcs-mollie-payment-gateway/releases/tag/v1.0.2

Any errors should now be logged in the module log.

Please follow all instructions (specially running composer) from the readme. I'm getting way to many people complaining it doesn't work while they didn't read 😅

**Comment by crtxz** [comment]
Hello,

Now i have this error in the log:
Unable to parse response as JSON

**Comment by mdeboer** [comment]
Ah, hmm. Then it seems Mollie sends a response that isn't valid JSON. Have you checked your Mollie API credentials in de module settings?

**Comment by crtxz** [comment]
yes, they are all correct. Even tried to reset them not working.
On a new client ID i get: 

Unable to connect to "https://api.mollie.nl/v1/customers/???;J???
-?Aͪ": 3 Illegal characters found in URL

**Comment by mdeboer** [comment]
Oh wow, that's odd and should not happen. I'll need to investigate further.
